From 080d3edd3d0338df0726b262ba73ca07631070cd Mon Sep 17 00:00:00 2001
From: Alejandro Sierra <b18039@freescale.com>
Date: Thu, 14 Jan 2016 11:35:29 -0600
Subject: [PATCH 16/37] ARM: dta: bus freq: disabled on IMX6DQSCM

Bus freq feature disabled on IMX6DQSCM due to some LPDDR2 issues.
- MXSCM-62: ARM: dts: bus freq: system hang after audio playback
when system in low bus freq state
- MXSCM-63: ARM: dts: bus freq: console didn't show bus freq change
info "Bus freq set to 24000000 start.." after system entered in low
bus freq state when 'echo 8 > /proc/sys/kernel/printk.

Signed-off-by: Alejandro Sierra <b18039@freescale.com>
---
 arch/arm/boot/dts/imx6dqscm-1gb-evb-ldo.dts | 9 +++++----
 arch/arm/boot/dts/imx6dqscm-qwks-rev1.dtsi  | 5 +++++
 2 files changed, 10 insertions(+), 4 deletions(-)

diff --git a/arch/arm/boot/dts/imx6dqscm-1gb-evb-ldo.dts b/arch/arm/boot/dts/imx6dqscm-1gb-evb-ldo.dts
index 540b58f..2635286 100644
--- a/arch/arm/boot/dts/imx6dqscm-1gb-evb-ldo.dts
+++ b/arch/arm/boot/dts/imx6dqscm-1gb-evb-ldo.dts
@@ -15,10 +15,11 @@
 / {
 	model = "Freescale i.MX6D SCM EVB";
 	compatible = "fsl,imx6q-sabresd", "fsl,imx6q";
-};
-
-&busfreq {
-        fsl,max_ddr_freq = <400000000>;
+        soc {
+            busfreq {
+                status = "disabled";
+            };
+        };
 };
 
 &ecspi1 {
diff --git a/arch/arm/boot/dts/imx6dqscm-qwks-rev1.dtsi b/arch/arm/boot/dts/imx6dqscm-qwks-rev1.dtsi
index a8c75dd..e559a0b 100644
--- a/arch/arm/boot/dts/imx6dqscm-qwks-rev1.dtsi
+++ b/arch/arm/boot/dts/imx6dqscm-qwks-rev1.dtsi
@@ -20,6 +20,11 @@
 		mxcfb3 = &mxcfb4;
 	};
 
+        soc {
+            busfreq {
+                status = "disabled";
+            };
+        };
 	hannstar_cabc {
 		compatible = "hannstar,cabc";
 
-- 
1.9.1

